nx/lib/nx.ex

```diff
@@ -4662,7 +4662,7 @@ defmodule Nx do
       iex> yv.vectorized_axes
       [y: 1, x: 1]
   """
-  @doc type: :shape
+  @doc type: :vectorization
```

Comment:
@josevalim I introduced this new category so that people can more easily find these 3 related functions. Let me know if you think we should keep it as :shape still

Reply:
agreed.